Dockerfile.deploy.rhel

```diff
@@ -0,0 +1,26 @@
+FROM prod.registry.devshift.net/osio-prod/base/pcp:latest
```

Comment:
There are small differences, and it might work if we remove L14-17 but to be on the safe side I let them go through. This is the dockerfile it's based on:
https://github.com/rhdt/EL-Dockerfiles/blob/master/base/pcp/Dockerfile